app/assets/stylesheets/application.css

```diff
@@ -23,24 +23,9 @@
    color: red;
  }
 
+/* css for error exclamation point */
  .excl {
   background-color: black;
   padding: 6px;
   border-radius: 100%;
  }
-
```

Comment:
What is the thinking behind breaking these out into separate files? I am not opposed to it, just curious about your reasoning.

Reply:
Once applications get large - having your CSS structured similarly to your views is helpful. This is just some preemptive organization.

Reply:
Cool. I support this. Just checking in with you two on reasoning.